ibm_catalog.json

```diff
@@ -0,0 +1,311 @@
+{
+  "products": [
+    {
+      "label": "[Add-ons Beta] Cloud automation for Observability Agents",
+      "name": "testing-deploy-arch-ibm-observability-agents",
+      "product_kind": "solution",
+      "tags": [
+        "ibm_created",
+        "logging_monitoring"
+      ],
+      "short_description": "Deploys IBM Cloud Observability agents to a cluster",
+      "long_description": "Solutions that support deploying IBM Cloud Observability agents for logging, monitoring, and activity tracking.",
+      "offering_docs_url": "https://github.com/terraform-ibm-modules/terraform-ibm-observability-agents/blob/main/README.md",
+      "offering_icon_url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-observability-da/main/images/observability-icon.svg",
+      "flavors": [
+        {
+          "label": "Standard",
+          "name": "standard",
+          "working_directory": "solutions/standard",
+          "licenses": [
+            {
+              "id": "LICENSE",
+              "name": "LICENSE",
+              "type": "text/plain",
+              "description": "LICENSE"
+            }
+          ],
+          "compliance": {},
+          "iam_permissions": [
+            {
+              "service_name": "containers-kubernetes",
+              "role_crns": [
+                "crn:v1:bluemix:public:iam::::serviceRole:Manager",
+                "crn:v1:bluemix:public:iam::::role:Viewer"
+              ]
+            }
+          ],
+          "architecture": {
+            "features": [
+              {
+                "title": "Works with any openshift or k8s cluster",
+                "description": "Works with the Red Hat OpenShift cluster."
+              },
+              {
+                "title": "Deploys the IBM Cloud Monitoring agent on an existing cluster.",
+                "description": "Deploys the IBM Cloud Monitoring agent on an existing Red Hat OpenShift cluster."
+              },
+              {
+                "title": "Deploys the IBM Logs agent on an existing cluster.",
+                "description": "Deploys the IBM Logs agent on an existing Red Hat OpenShift cluster."
+              }
+            ],
+            "diagrams": [
+              {
+                "diagram": {
+                  "url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-observability-agents/refs/heads/addons/reference-architecture/deployable-architecture-observability-agents.svg",
+                  "caption": "Observability agents on a Red Hat OpenShift cluster",
+                  "type": "image/svg+xml"
+                },
+                "description": "Observability agents deployed on a Red Hat OpenShift cluster on a secure landing zone."
+              }
+            ]
+          },
+          "dependencies": [
```

Comment:
I don't think we should wire up any inputs yet - the feature is not stable enough. Suggest to just list the dependencies with no wiring. Same as what is [here](https://github.com/terraform-ibm-modules/terraform-ibm-observability-da/blob/fc797a01328c5da3bab8096384fc6b8713b08db2/ibm_catalog.json#L529C1-L550C13)